Lib/compression/zstd/__init__.py

```diff
@@ -11,6 +11,7 @@
     'get_frame_info',
     'Strategy',
     'train_dict',
+    'zstd_version_info',
```

Comment:
Why moving this? it's better to keep it close to `zstd_version`.

Reply:
Reverted! You're right, it was a mistake to include it here, I got mixed up here.